lib/channels/src/index.ts

```diff
@@ -60,9 +58,8 @@ export class Channel {
   }
 
   addPeerListener(eventName: string, listener: Listener) {
-    const peerListener = listener;
-    peerListener.ignorePeer = true;
-    this.addListener(eventName, peerListener);
+    console.warn('channel.addPeerListener is deprecated');
```

Comment:
Please use `util-deprecate` for consistency

Reply:
done